src/jsc/jsc_rpc_network_transport.hpp

```diff
@@ -18,7 +18,7 @@
 
 #pragma once
 
-#include "sync/generic_network_transport.hpp"
+#include "realm/object-store/sync/generic_network_transport.hpp"
```

Comment:
Is this a pattern that we need to repeat elsewhere?

Reply:
I don't understand the question? Do you see an issue here?

Reply:
No, but i was wondering if there was a reason this is the only place you're making the change. I can look into this myself afterwards. Just curious.

Reply:
that's cause this file changed its location in the monorepo